chore: fix some comments (#1146)
projectcapsule/capsule: CONTRIBUTING.md

2. **Create a branch:**

> Create a new breach and navigate to the branch using this command.

Reviewer: ```suggestion
Create a new branch and navigate to it using this command.
```
